modules/system/assets/ui/js/tab.js

```diff
@@ -39,6 +39,20 @@
             self.closeTab($(ev.target).closest('ul.nav-tabs > li, div.tab-content > div'), force)
         })
 
+        this.$el.on("mousedown", "li[data-tab-id]", function (ev, data) {
+            if (2 == ev.which) {
```

Comment:
How about instead of reimplementing the close tab functionality here, you just fire the `close.oc.tab` event and let https://github.com/octobercms/october/blob/master/modules/system/assets/ui/js/tab.js#L36-L40 handle the closing logic?

Reply:
Yup that sounds good!

Reply:
Pushed a commit a few minutes ago, it makes 'close.oc.tab' event to be fired and also the function that handles such event, now handles the logic for middle mouse click tab close functionality also.

Reply:
@AxDSan I don't see the commit yet, did you push it to the correct branch?

Reply:
Ooh i think i messed up, I’ll check back when I get home.

Reply:
Ok forgot to actually push it xD